Doesn't build with latest mpv since 5.7.6.109

```
In file included from src/backends/mpv/mpv_proxy.cpp:32:
src/backends/mpv/mpv_proxy.h:40:10: fatal error: mpv/qthelper.hpp: No such file or directory
   40 | #include <mpv/qthelper.hpp>
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
```

Looks similar to https://github.com/easymodo/qimgv/issues/184, which suggests that the header was deprecated ages ago. Please consider avoid its usage or include the header in the repo like qimgv did.

--- a/src/backends/mpv/mpv_glwidget.h
+++ b/src/backends/mpv/mpv_glwidget.h
@@ -35,7 +35,7 @@
 #include <mpv/render_gl.h>
 #include <mpv_proxy.h>
 #undef Bool
-#include <mpv/qthelper.hpp>
+#include "../../vendor/qthelper.hpp"
 #include <DGuiApplicationHelper>
 //DWIDGET_USE_NAMESPACE
 

--- a/src/backends/mpv/mpv_proxy.h
+++ b/src/backends/mpv/mpv_proxy.h
@@ -37,7 +37,7 @@
 #include <player_engine.h>
 #include <xcb/xproto.h>
 #undef Bool
-#include <mpv/qthelper.hpp>
+#include "../../vendor/qthelper.hpp"
 
 typedef mpv_event *(*mpv_waitEvent)(mpv_handle *ctx, double timeout);
 typedef int (*mpv_set_optionString)(mpv_handle *ctx, const char *name, const char *data);

--- a/src/libdmr/compositing_manager.cpp
+++ b/src/libdmr/compositing_manager.cpp
@@ -46,7 +46,7 @@
 #include <GL/glx.h>
 #include <GL/glxext.h>
 #undef Bool
-#include <mpv/qthelper.hpp>
+#include "../vendor/qthelper.hpp"
 
 typedef const char *glXGetScreenDriver_t (Display *dpy, int scrNum);
 

